<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CentreControll</class>
 <widget class="QWidget" name="CentreControll">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="btnMusic">
     <property name="text">
      <string>音乐播放器</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSmart">
     <property name="text">
      <string>智能家具</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnWeather">
     <property name="text">
      <string>天气预报</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn">
     <property name="text">
      <string>监控</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnBack">
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
